specs/pvs/corla.pvs

```diff
@@ -372,8 +416,12 @@ BEGIN
 % @todo Each scanner has a unique id number.
   vvpat: TYPE
   verified?: TYPE = pred[storage_container]
-  log: TYPE
-  chain_of_custody_log: TYPE FROM log
+
+%%% Voting system's logs are precisely defined in federal
+%%% standards. Those standards are known as the ``Voluntary Voting
+%%% System Guidlines'', or VVSG for short.
```

Comment:
I've been involved in the VVSG update discussions on event logging, and they are good on syntax, but near worthless on semantics. The adopted VVSG standards do give some precise definitions, but I don't know if they cover anything relevant for us.